test/src/e2e/customAction.test.ts

```diff
@@ -42,27 +45,33 @@ describe("customAction", function() {
 
     describe("customAction", function() {
         it("should get initial state", async function() {
-            const actionData = await node.sdk.rpc.engine.getCustomActionData(
-                hitActionHandlerId,
-                ["hit count"]
+            const actionData = await node.rpc.engine.getCustomActionData({
+                    handlerId: hitActionHandlerId,
+                    bytes: `0xca${hitcount}`,
```

Comment:
Why the `ca` is needed? Do you know that?

Reply:
It seems that toHex(RLP.encode(["hit count"])) == `0xca${toHex(RLP.encode("hit count"))}`. Could you check it?